### PR TITLE
Second round deliberation panel

### PR DESCRIPTION
**Pull request by vitorvicente**
Second round deliberation panel for voting/configuration

### CODE REVIEW

File: src/components/Deliberation/InterviewerView.js [hunk 1]
```diff
@@ -278,7 +322,12 @@ class InterviewerView extends Component {
             const accepted = positiveVotes / allVotes.length >= 0.75;
             if (accepted) {
               const ref = this.props.firebase.application(id);
-              batch.update(ref, { "deliberation.accepted": true });
+              batch.update(ref, {
```
Comment (ROODAY) [inline src/components/Deliberation/InterviewerView.js:325]: Will we always need to set this data about provisional stuff here? This seems to be the function for just sending results in general, perhaps there should be a conditional gating which data is added to the update? Or is there a reason to always add the data here?

File: src/components/Deliberation/SecondRound.js [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    const ApplicantStatus = ({
+      provisional: { contribution, meetings },
+      name,
+      id,
+    }) => (
+      <tr>
+        <td>{name}</td>
+        <td>{contribution ? "Complete" : "Not Complete"}</td>
+        <td>{meetings ? "Complete" : "Not Complete"}</td>
+        {!settings.deliberationsOpen && (
```
Comment (ROODAY) [inline src/components/Deliberation/SecondRound.js:46]: Just double checking, is this conditional correct or is this inversed for testing?

File: src/components/Deliberation/SecondRound.js [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+
+    return (
+      <Container flexdirection="column">
+        <Table bordered hover>
+          <thead>
+            <tr>
+              <th>Name</th>
+              <th>Contribution Status</th>
+              <th>Meetings Status</th>
+              {!settings.deliberationsOpen && <th>Edit</th>}
```
Comment (ROODAY) [inline src/components/Deliberation/SecondRound.js:73]: Double checking again, is this conditional correct or is this inversed for testing?

File: src/components/Deliberation/SecondRound.js [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+            </tr>
+          </thead>
+          <tbody>
+            {provisionalMemberApplications.map((application) => (
+              <ApplicantStatus key={application.id} {...application} />
+            ))}
+          </tbody>
+        </Table>
+
+        {!settings.deliberationsOpen && (
```
Comment (ROODAY) [inline src/components/Deliberation/SecondRound.js:83]: Double checking again, is this conditional correct or is this inversed for testing?

**Review by ROODAY — CHANGES_REQUESTED** [review]

File: src/components/Deliberation/InterviewerView.js (hunk 1 above)
Comment (vitorvicente) [reply src/components/Deliberation/InterviewerView.js]: Fixed in 32ec323b07c251c4abd46a6c3c39198d223fd440

**Review by ROODAY — APPROVED** [review]

**Pull request merged**